spec/models/category_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```

Comment:
- [ ] You only added tests for your models. Please add request specs for your project as well so that you could cover most of the components in your project 👍

Reply:
- [ ] Please add more tests as requested by the previous reviewer